flax/linen/attention.py

```diff
@@ -58,6 +59,9 @@ def dot_product_attention_weights(query: Array,
       shape `[batch..., num_heads, q_length, kv_length]`.
       This can be used for incorporating causal masks, padding masks,
       proximity bias, etc.
+    mask: mask for the attention weights. This should be broadcastable to the
```

Comment:
Can you specify expected values? Is the mask all 0's and 1's?

Reply:
done